lib/matplotlib/cbook/__init__.py

```diff
@@ -1431,7 +1431,7 @@ def _reshape_2D(X, name):
         raise ValueError("{} must have 2 or fewer dimensions".format(name))
 
 
-def violin_stats(X, method, points=100):
+def violin_stats(X, method, percentiles=None, points=100):
```

Comment:
Could you add the new keyword argument to the end of the signature instead, so people calling it like `(X, method, 100)` with no keywords don't have to change their code.